tests/DefaultSpec.hs

```diff
@@ -0,0 +1,39 @@
+-- |
+-- Module      : DefaultSpec
+-- Description : Ensure that deriving via (Default  a)newtype works
+-- License     : BSD-3-Clause
+--
+-- Maintainer  : generics@haskell.org
+-- Stability   : experimental
+-- Portability : non-portable
+--
+-- Tests DerivingVia on GHC versions 8.6 and above.
+{-# OPTIONS_GHC -fno-warn-unused-top-binds #-}
```

Comment:
Let's not do this. Instead, just export everything from this module. That is, change:

```haskell
module DefaultSpec (main, spec) where
```

To:

```haskell
module DefaultSpec where
```